docs/guides/server/hostname.adoc

```diff
@@ -85,42 +85,276 @@ To reduce attack surface, the administration endpoints for Keycloak and the Admi
 Therefore, you can secure them by using a reverse proxy.
 For more information about which paths to expose using a reverse proxy, see <@links.server id="reverseproxy"/>.
 
-== Example Scenarios
-The following are more example scenarios and the corresponding commands for setting up a hostname.
 
-Note that the `start` command requires setting up TLS. The corresponding options are not shown for example purposes. For more details, see <@links.server id="enabletls"/>.
+== Configuration Examples by Different Scenarios
 
-=== Exposing the server behind a TLS termination proxy
 
-In this example, the server is running behind a TLS termination proxy and publicly available from `https://mykeycloak`.
+Through this set of hostname configuration options, Keycloak aims to provide a flexible way to adapt to the characteristics of different environments.
 
-.Configuration:
-<@kc.start parameters="--hostname=mykeycloak --proxy=edge"/>
+Misconfiguration of the Hostname and/or its Reverse Proxy options can lead to unusual and unpleasant behavior, such as Keycloak's Administration Console not loading correctly.
 
-=== Exposing the server without a proxy
+In an attempt to further improve the user experience and minimize the occurrence of misconfigurations, see the list of sample Hostname and Proxy configurations below, which is oriented to possible scenarios that may exist in different computing environments.
 
-In this example, the server is running without a proxy and exposed using a URL using HTTPS.
+The following are more example scenarios and the corresponding commands for setting up the `Hostname` and/or `Proxy` options.
 
-.Keycloak configuration:
-<@kc.start parameters="--hostname-url=https://mykeycloak"/>
+> Note that the `start` command requires setting up TLS. The corresponding options are not shown for example purposes. For more details, see <@links.server id="enabletls"/>.
+
+
+=== Scenario 01 - Beginner/Quick Test in Localhost with HTTP only
+
+* __**Description**__:
+  ** The most basic option to run, in localhost, all default values.
+* __**Recommended Settings / Keycloak configuration**__:
+
+  <@kc.startdev parameters="" />
+
+* __**Expected Behavior**__:
+  ** Both Frontend/Backend and Admin Console at: http://localhost:8080
+
+---
```

Comment:
Ok. In the regard of simplifying the format, suggestions accepted and applied. Thanks.

- `remove the slash from every place, where it doesn't fulfil the role of an alternative - scenario/example; recommended settings/keycloak configuration; beginner/quick - all have very similar meaning in this context,`

  - Done. Please review again If it is better this way.
  
- `on the other hand, let's add more precise, more detailed descriptions; use rather full sentence instead of collocations separated by commas`
 
  - What is your suggestion of full sentence, where do you think it is not good?

- `remove the bullet points - if you need, break the line instead`

  - Done.
  
- `use either bold or italic text (not both at the same time).`

  - Done.
  
- `Regardless, I'm not sure, whether it's worth to mention this particular scenario, as it doesn't provide any additional information regarding the hostname setup (compared to others bellow). But it's up to a discussion, I used it as a reference example of how the format could be potentially improved.`  

  - Exactly. In my view, the most important points of this review are: Evaluate which scenarios make sense to mention in this Guide and If the `kc start` command options are right for each scenario. And I need your help in this effort.
  
Thanks.

Reply:
> What is your suggestion of full sentence, where do you think it is not good?

I meant it rather like a reminder that we should avoid sentence structures like the one you use initially:
`The most basic option to run, in localhost, all default values.`

Anyway, I think that right now the rest is ok. Thanks!